src/controllers/overseas.name.controller.ts

```diff
@@ -1,52 +1,69 @@
 import { NextFunction, Request, Response } from "express";
 import { Session } from "@companieshouse/node-session-handler";
-
 import * as config from "../config";
 import { ApplicationData } from "../model";
 import { logger } from "../utils/logger";
 import { isActiveFeature } from "../utils/feature.flag";
-import { getApplicationData, setExtraData } from "../utils/application.data";
 import { postTransaction } from "../service/transaction.service";
+import { getApplicationData, setExtraData } from "../utils/application.data";
 import { createOverseasEntity, updateOverseasEntity } from "../service/overseas.entities.service";
-import { EntityNameKey, OverseasEntityKey, Transactionkey } from "../model/data.types.model";
-import { getUrlWithParamsToPath, getUrlWithTransactionIdAndSubmissionId, transactionIdAndSubmissionIdExistInRequest } from "../utils/url";
+
+import {
+  EntityNameKey,
+  Transactionkey,
+  OverseasEntityKey,
+} from "../model/data.types.model";
+
+import {
+  getUrlWithParamsToPath,
+  getUrlWithTransactionIdAndSubmissionId,
+  transactionIdAndSubmissionIdExistInRequest
+} from "../utils/url";
 
 export const get = async (req: Request, res: Response, next: NextFunction) => {
+
   try {
+
     logger.debugRequest(req, `GET ${config.OVERSEAS_NAME_PAGE}`);
-    const appData: ApplicationData = await getApplicationData(req.session);
 
+    const appData: ApplicationData = await getApplicationData(req);
```

Comment:
A fair few formatting changes and other miscellaneous clean-ups. Is this the only 'real' change to the functionality?

Also, does it - or rather should it - have a corresponding unit test change? I'd have though we mock out the call to `getApplicationData`, so wondering how existing tests are passing if param has changed.

Reply:
- The 'real' change is that the call to `getApplicationData` is  passing in the full request and not the session
- No need to add a unit test coz it's already covered by the mock